src/ast/passes/codegen_llvm.cpp

```diff
@@ -96,12 +96,16 @@ CodegenLLVM::CodegenLLVM(Node *root, BPFtrace &bpftrace)
                          llvm::DEBUG_METADATA_VERSION);
 
   // Set license of BPF programs
-  const std::string license = "GPL";
+  const auto &license = bpftrace_.get_license();
   auto license_var = llvm::dyn_cast<GlobalVariable>(module_->getOrInsertGlobal(
       "LICENSE", ArrayType::get(b_.getInt8Ty(), license.size() + 1)));
   license_var->setInitializer(
       ConstantDataArray::getString(module_->getContext(), license.c_str()));
   license_var->setSection("license");
+
+  // XXX: Set licenses for manually loaded programs. Remove once libbpf is used
```

Comment:
what is XXX?

Reply:
Just to draw attention to this line in the future because it shouldn't stay permanently.

Basically this: https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

But also because it gets syntax highlighted in my editor :P

Reply:
TIL!